### DISCUSSION TITLE
Add Studio 1.10 features to OrcaSlicer soon?

### DISCUSSION THREAD

**Discussion by LJKraus** [OrcaSlicer/OrcaSlicer#7131] [Ideas]
It is still a beta release, but Studio 1.10 has a number of additions that users have been requesting (and a few that were taken from Orca).  I'm looking forward to inset ironing and [stabilizing supports](https://github.com/SoftFever/OrcaSlicer/discussions/6988#discussion-7267878) on tall thin vertical supports.  The ability to bind a printer on a different subnet is also welcome - I don't need that feature but complaints about it have been common.  It would be nice if these features could be incorporated into OrcaSlicer

**Comment by rocket59** [comment]
Seconded. Bambu has just released updated firmware (on the P1 series anyway) to address issues with Studio 1.10 and subnet binding. If this fixes them this should be straightfoward to incorporate into Orca?
